src/c++/perf_analyzer/client_backend/mock_client_backend.h

```diff
@@ -210,20 +210,44 @@ class MockClientStats {
 
   void SetDelays(std::vector<size_t> times)
```

Comment:
Can you add a comment explaining the requirements for a valid input? How many elements should `times` have? As many as requests are expected to be made?

Reply:
done